README.rst

```diff
@@ -41,6 +46,11 @@ Authors
 The first release of FermiLib (v0.1a0) was developed by `Ryan Babbush <https://research.google.com/pubs/RyanBabbush.html>`__, `Jarrod McClean <https://crd.lbl.gov/departments/computational-science/ccmc/staff/alvarez-fellows/jarrod-mcclean/>`__, `Damian S. Steiger <http://www.comp.phys.ethz.ch/people/person-detail.html?persid=165677>`__, `Ian D. Kivlichan <http://aspuru.chem.harvard.edu/ian-kivlichan/>`__, `Thomas
 Häner <http://www.comp.phys.ethz.ch/people/person-detail.html?persid=179208>`__, `Vojtech Havlicek <https://github.com/VojtaHavlicek>`__, `Matthew Neeley <https://maffoo.net/>`__, and `Wei Sun <https://github.com/Spaceenter>`__.
 
+Questions?
+----------
+
+If you have any other questions, please contact fermilib at projectq.ch.
```

Comment:
Yeah I made the same comment but somebody *cough* @idk3 *cough* was worried that bots would find the email address and spam us. But I don't think it is too common to crawl GitHub for this sort of thing. Ian, please change (or contest) and then write something so I know you made the change. I will then merge the PR.